yourgov/components/FormMobileFoodVendorPermit/FormState.ts

```diff
@@ -36,7 +36,7 @@ export type FormStep<StepNumber extends AnyStepNumber = AnyStepNumber> = {
 	}>;
 };
 
-export type AnyStepNumber = TaskStepNumber;
+export type AnyStepNumber = StepNumber;
```

Comment:
😅

Reply:
lol it used to be `export type AnyStepNumber = Task1StepNumber; // | Task2StepNumber | Task3StepNumber;` before the previous PR… which you approved 😉

Reply:
🤣

Reply:
I have pushed a change to remove this